setup.py

```diff
@@ -105,14 +102,12 @@
 ]
 
 install_requires = [
-    'Flask>=1.0.4',
-    'invenio-app>=1.2.3,<1.3.0',
-    'invenio-base>=1.2.0,<1.3.0',
-    'invenio-cache>=1.0.0,<1.1.0',
-    'invenio-celery>=1.1.1,<1.2.0',
-    'invenio-config>=1.0.2,<1.1.0',
-    'invenio-i18n>=1.1.1,<1.2.0',
-    'Werkzeug>=0.15.0,<1.0.0'
+    'invenio-app>=1.2.6,<1.3.0',
```

Comment:
comment: it is okay. **must not be bumped**, invenio-app 1.3.0+ contains semantic-ui work.